_data/toc.yaml

```diff
@@ -1814,6 +1806,35 @@ manuals:
   - path: /compose/release-notes/
     title: Release notes
 
+- sectiontitle: Docker CLI
```

Comment:
Should be placed right after `Docker Engine` section?

Reply:
Yeah maybe, it looks strange in the current position tbh. The whole TOC tree is getting a bit crowded... We'll need to review it for a bigger restructure soon I think (IMO - break out topics that aren't related to software development, e.g. admin etc)